config/nrfconnect/Kconfig

```diff
@@ -18,6 +18,30 @@ mainmenu "CHIP application"
 
 source "Kconfig.zephyr"
 
+config CHIP
+	bool "Enable CHIP"
+	default n
+	select CPLUSPLUS
+	select REBOOT
```

Comment:
imply REBOOT. native_posix may not support REBOOT. Also maybe add "imply LIB_CPLUSPLUS" here?

Reply:
Done